roles/upgrade/README.md

```diff
@@ -18,6 +21,15 @@ The `upgrade` role is used to kick off the Leapp OS upgrade on the target host.
 
 See [`upgrade.yml`](../../playbooks/upgrade.yml).
 
+```ansible
+- name: Perform OS upgrade
+  ansible.builtin.import_role:
+    name: infra.leapp.upgrade
+```
```

Comment:
Let just link to the example playbook rather than adding this and giving two different examples... 
```suggestion
```